tweet_on_upcoming_and_recently_released_movies/main.bal

```diff
@@ -0,0 +1,61 @@
+import ballerina/io;
+import ballerinax/themoviedb;
+import ballerinax/azure_openai.text;
+import ballerina/http;
+import ballerinax/twitter;
```

Comment:
i think these should be arranged in alphabetical order and follow the guidelines in
https://learn-ballerina.github.io/best_practices/format_the_code.html#order-of-the-imports-in-a-bal-file

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/63/commits/7c756cfcba039786cbdf07ee3fe1a498746b3d32.